client/src/pages/Signup/SignupParents/parentSignupService.ts

```diff
@@ -8,7 +8,7 @@ interface IAxiosResponse {
 export const parentSignup = async (data: FieldValues): Promise<void> => {
   try {
     const { termsAndConditions, confirmPassword, ...parentDetails } = data;
-    await axios.post(
+    axios.post(
```

Comment:
you need to write the word "return".
axios.post still returns nothing so it will still be Promise<void>